src/els_range.erl

```diff
@@ -17,7 +17,21 @@ compare( #{from := FromA, to := ToA}
 compare(_, _) ->
   false.
 
--spec range(pos(), poi_kind(), any(), any()) -> poi_range().
+-spec range(pos() | {pos(), pos()}, poi_kind(), any(), any()) -> poi_range().
+range({{Line, Column}, {ToLine, ToColumn}}, Name, _, _Data)
```

Comment:
There were cases (e.g. folding ranges) in which we wanted to provide both the beginning and the end of the range from the upstream logic so it made sense to have the creation here instead of sometimes calling `els_poi:new/[3,4]` from `els_parser` and sometime relying on the logic of `els_range:range/4`.